sdk/signalr/Microsoft.Azure.Management.SignalR/src/Generated/ISignalROperations.cs

```diff
@@ -47,7 +47,7 @@ public partial interface ISignalROperations
         /// <exception cref="Microsoft.Rest.ValidationException">
         /// Thrown when a required parameter is null
         /// </exception>
-        Task<AzureOperationResponse<NameAvailability>> CheckNameAvailabilityWithHttpMessagesAsync(string location, NameAvailabilityParameters parameters = default(NameAvailabilityParameters), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<AzureOperationResponse<NameAvailability>> CheckNameAvailabilityWithHttpMessagesAsync(string location, NameAvailabilityParameters parameters, Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
Breaking change

Reply:
Hey Markcowl, thanks for the cr, csproj and assembly file is modified. please help take a look